packages/quantic/cypress/integration/tab/tab.cypress.ts

```diff
@@ -98,4 +100,31 @@ describe('quantic-tab', () => {
       Expect.activeTabContains(tabs.knowledge.label);
     });
   });
+
+  describe('when using accessibility', () => {
+    beforeEach(() => {
+      loadFromUrlHash({}, `tab=${tabs.all.label}`);
+    });
+
+    it('when SpaceKey is pressed', () => {
+      TabSelectors.button().first().focus().type(' ');
+      Expect.activeTabContains(tabs.all.label);
+    });
+
+    it('when TabKey && SpaceKey is pressed', () => {
+      TabSelectors.button().first().focus().tab().type(' ');
```

Comment:
I think most or all of these `.focus().tab().type(' ')` kind of sequences could be added to `tab-actions.ts`

Reply:
moved to .action.ts file